internal/api/oidc/client.go

```diff
@@ -253,6 +263,13 @@ func (o *OPStorage) GetPrivateClaimsFromScopes(ctx context.Context, userID, clie
 	if len(userMetaData) > 0 {
 		claims = appendClaim(claims, ClaimUserMetaData, userMetaData)
 	}
+	resourceOwner, err := o.assertUserResourceOwner(ctx, userID)
+	if err != nil {
+		return nil, err
+	}
+	if len(resourceOwner) > 0 {
+		claims = appendClaim(claims, ClaimResourceOwner, resourceOwner)
+	}
```

Comment:
this would append the resourceowner claim even if I don't provide the scope, right?

shouldn't it be checked in the for loop at the beginning of the function?
I overlooked that for the userMetadata, too.

Reply:
changed it for both